src/components/DefaultPageLayout.tsx

```diff
@@ -69,13 +68,6 @@ const DefaultPageLayout = ({
   const isSaveDialogClosed = useStore((s) => s.save.step === SaveStep.None);
 
   const toast = useToast();
-  const { appNameFull } = useDeployment();
-
-  useEffect(() => {
-    document.title = titleId
-      ? `${intl.formatMessage({ id: titleId })} | ${appNameFull}`
-      : appNameFull;
-  }, [appNameFull, intl, titleId]);
```

Comment:
If we pick this PR up again, then we need this or something equivalent back or the page title doesn't change when you navigate. The text for the page title itself is translated.